src/org/jointheleague/graphical/robot/curves/Quad.java

```diff
@@ -6,7 +6,7 @@
 import java.awt.geom.Path2D;
 import java.awt.geom.QuadCurve2D;
 
-final public class Quad implements Drawable, Segment {
+public final class Quad implements Drawable, Segment {
```

Comment:
True. This just was just a SonarLint recommendation i decided to go with: 

> The Java Language Specification recommends listing modifiers in the following order:
> 
> 1. Annotations
> 
> 2. public
> 
> 3. protected
> 
> 4. private
> 
> 5. abstract
> 
> 6. static
> 
> 7. final
> 
> 8. transient
> 
> 9. volatile
> 
> 10. synchronized
> 
> 11. native
> 
> 12. strictfp
> 
> Not following this convention has no technical impact, but will reduce the code's readability because most developers are used to the standard order.

Reply:
OK. I agree, put public before final